compiler/lib/eval.ml

```diff
@@ -127,8 +137,15 @@ let eval_prim x =
   | Extern name, l -> (
       let name = Primitive.resolve name in
       let (module Int : Int) =
-        match Config.target () with
-        | `JavaScript -> (module Int32)
+        match target with
+        | `JavaScript ->
+            (module struct
+              include Int32
+
+              let of_int32_warning_on_overflow = Fun.id
+
+              let to_int32 = Fun.id
+            end)
```

Comment:
One should define this module at toplevel rather than allocating a module each time.

Reply:
Done